internal/icinga2/api_responses.go

```diff
@@ -10,7 +10,7 @@ import (
 // UnixFloat is a custom time.Time type for millisecond Unix timestamp, as used in Icinga 2's API.
 type UnixFloat time.Time
 
-// Time returns the time.Time of UnixFloat.
+// Time returns the time.Time of UnixFloat or zero value of time.Time.
```

Comment:
That doesn't really add any information for me. After all, this function is only a glorified cast, is returns the zero value if the underlying `Time` is set to the zero value. I think the idea behind this might have been to say that JSON `0` is mapped to `Time{}`, but that would better fit into a comment for the `UnmarshalJSON` method (including a short explanation why it does so).